authors/views.py

```diff
@@ -231,95 +281,97 @@ def list(self, request, *args, **kwargs):
-#
-#     def put(self, request, *args, **kwargs):
-#         if self.relation_check():
-#             return Response({
-#                 'following_relation_exist': 'True',
-#                 'following_relation_put': 'False'
-#             })
-#         else:
-#             try:
-#                 usernames = self.usernames()
-#                 FollowerCount.objects.create(follower=usernames[0],
-#                                              user=usernames[1])
-#                 return Response({
-#                     'following_relation_exist': 'False',
-#                     'following_relation_put': 'True'
-#                 })
-#             except:
-#                 return Response({
-#                     'following_relation_exist': 'False',
-#                     'following_relation_put': 'False'
-#                 })
-#
-#     def delete(self, request, *args, **kwargs):
-#         if self.relation_check():
-#             usernames = self.usernames()
-#             relation = FollowerCount.objects.filter(
-#                 follower=usernames[0]).filter(user=usernames[1])
-#             try:
-#                 relation.delete()
-#                 return Response({
-#                     'following_relation_exist': 'True',
-#                     'following_relation_delete': 'True'
-#                 })
-#             except:
-#                 return Response({
-#                     'following_relation_exist': 'True',
-#                     'following_relation_delete': 'False'
-#                 })
-#         else:
-#             return Response({
-#                 'following_relation_exist': 'False',
-#                 'following_relation_delete': 'False'
-#             })
+class FollowersAPIView(ListAPIView):
+    """ GET an Author's all followers """
+
+    # serializer_class = serializers.FollowersSerializer
+    # renderer_classes = (renderers.FollowersRenderer,)
+    http_method_names = ['get']
+    def list(self, request, *args, **kwargs):
+        queryset = []
+        uuid = self.kwargs['author']
+        author = Author.objects.filter(uuid=uuid).first()
+        follower_lst = Followers.objects.filter(user=author).first()
+        serializer = serializers.AuthorSerializer(follower_lst.items, many=True)
+        response = {"type": "followers", "items": serializer.data}
+        return Response(response)
+
+
+class FollowerAPIView(RetrieveUpdateDestroyAPIView):
+    """ GET if is a follower PUT a new follower DELETE an existing follower"""
+
+    serializer_class = serializers.FollowersSerializer
+    renderer_classes = [JSONRenderer]
+
+    # def usernames(self, *args, **kwargs):
+    #     follower_uuid = self.kwargs['another_author']
+    #     follower = Author.objects.filter(uuid=follower_uuid).first()
+    #     user_uuid = self.kwargs['author']
+    #     user = Author.objects.filter(uuid=user_uuid).first()
+    #     return follower.username, user.username
+    #
```

Comment:
Is this chunk of comments required? If not might be good to remove it clean up bloat.